pheget/static/js/region.js

```diff
@@ -3,6 +3,58 @@
 const API_BASE = 'https://portaldev.sph.umich.edu/api/v1/';
 // Set to smaller values for testing; go up to 50k or 200k after we make it more efficient
 const MAX_EXTENT = 500000;
+
+
+LocusZoom.TransformationFunctions.set('pip_yvalue', function (x) {return Math.max(Math.log10(x), -6);});
+
+var retrieveBySuffix = function(input, suffix) {
```

Comment:
I've refactored this function and moved duplicate transformation functions to common.js.

Oddly, the duplicate code was implemented in very slightly different ways across region vs variant.js. (eg whether it checked for null values; it wasn't clear to me if these values were allowed to be null)

@pjvandehaar , feel free to make adjustments if I missed an edge case.